### PR TITLE
Bug 1867821: Include spec.priority for default sources

### PR DESCRIPTION
**Pull request by anik120**
**Description of the change:**
This PR introduces spec.priority for the default CatalogSource introduce in #318 

**Motivation for the change:**

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

**Comment by anik120** [comment]
/hold 
Blocked by #323

File: cmd/manager/main.go [hunk 1]
```diff
@@ -13,15 +13,14 @@ import (
 	"github.com/operator-framework/operator-marketplace/pkg/migrator"
 
 	apiconfigv1 "github.com/openshift/api/config/v1"
-	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	ofapi "github.com/operator-framework/api/pkg/operators/v1alpha1"
```
Comment (exdx) [inline cmd/manager/main.go:16]: nit: I think the alias of the package should be `v1alpha1` or `operatorsv1alpha1` per our import convention in OLM and k8s in general. See the OLM e2e tests for examples of this. For example, in the catalog source e2e tests catalogs are simply `v1alpha1.CatalogSource`

Comment (anik120) [reply cmd/manager/main.go:16]: @exdx I wanted to distinguish between marketplace v1 and v2 and olm's v1alpha1. I think `operatorsv1alpha1` does the job too so I'll switch this over to `operatorsv1alpha1`

**Comment by anik120** [comment]
/hold cancel 

Ready for review.

**Comment by anik120** [comment]
Closing in favor of #329

**Pull request closed without merging**